database/index.js

```diff
@@ -0,0 +1,59 @@
+const mysql = require('mysql');
+const mysqlConfig = require('./config.js');
+
+const connection = mysql.createConnection(mysqlConfig);
+
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min)) + min; //The maximum is exclusive and the minimum is inclusive
+}
+
+function generateUsername() {
+  var text = "";
+  var possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
```

Comment:
Nice job using strings to assign rather than objects/arrays which may take up more space.
Would update the 'possible' variable name to be slightly more specific (ie. 'possibleChars'). Also super nit-picky.